helm/messenger/conf/nginx.conf

```diff
@@ -36,6 +36,13 @@ map $http_upgrade $connection_upgrade {
   default  upgrade;
   ""       close;
 }
+# TODO: Remove, when `drift` works on Safari with COEP set to `require-corp`:
+#       https://github.com/simolus3/drift/issues/2812#issuecomment-2122086577
+# Declares variable to set `Cross-Origin-Embedder-Policy` header with.
+map $http_user_agent $coep {
+    default                            require-corp;
+    ~(^(?!.*(?:Chrome|Edge)).*Safari)  credentialless;
+}
```

Comment:
@tyranron, `drift` seems to fail to initialize in Safari, when application is deployed with COEP set to `require-corp`. Not sure, if that's a bug in the `drift` package, `sqlite3` or WebKit, yet seems to be strange, as COEP set to `credentialless` works fine! Every other browser works perfectly fine with COEP set to `require-corp`, this is only related to Safari.

I've wrote a comment with the details in the `drift` repository for someone to take a look at this as well, perhaps I am missing something: https://github.com/simolus3/drift/issues/2812#issuecomment-2122086577

Thus I suggest to set COEP to `credentialless` for Safari temporary and creating a separate issue to track this problem.

Reply:
UPD. Not sure, but might be related: https://github.com/SamVerschueren/webkit-coep-disk-cache